fix candra

Shreeshrii opened the pull request.

akprasad commented:
Looking at this now. Thanks for your patience.

akprasad commented on README.md:
```diff
@@ -71,3 +71,19 @@ Adding a new scheme is simple:
+
+`make` - will start the webserver on port `9732`
+
+Open your browser and launch this url `http://127.0.0.1:9732/test/index.html` and see that all tests are ok.
```
I don't see why this is necessary. Can you explain? The `localhost`/qunit approach that exists already seems fine.

akprasad commented on README.md:
```diff
@@ -71,3 +71,19 @@ Adding a new scheme is simple:
+
+Open your browser and launch this url `http://127.0.0.1:9732/test/index.html` and see that all tests are ok.
+
+Changelog
```
No need for changelog, Git tracks that all already.

akprasad commented on sanscript/sanscript.js:
```diff
@@ -5,14 +5,29 @@
  * other Indian languages only incidentally.
  *
  * Released under the MIT and GPL Licenses.
+ *
```
See comment above about changelog.

akprasad commented on sanscript/sanscript.js:
```diff
@@ -33,16 +48,56 @@
+            accent: ['\u2060', '\u2060', "\u2060", "\u2060"],
+            combo_accent: ["", "", "", ""]
+        },
+		/* Grantha
```
nit: line up comment with ones below.

Shreeshrii commented:
Arun,

The Readme changes are by Naresh, I had forked his repo for use with
sanskrit documents.

Not being familiar with git, GitHub I was keeping a changelog within the js
script for my reference.

I think it would be easiest if you can include the relevant changes in your
repo and ignore what is unnecessary.

Thanks!


On 29-Aug-2017 9:32 AM, "Arun Prasad" <notifications@github.com> wrote:

> *@akprasad* commented on this pull request.
>
> Thanks for these changes, and thanks again for your patience.
> ------------------------------
>
> In README.md
> <https://github.com/sanskrit/sanscript.js/pull/7#discussion_r135688336>:
>
> > @@ -71,3 +71,19 @@ Adding a new scheme is simple:
>      Sanscript.addRomanScheme(schemeName, schemeData);
>
>  For help in creating `schemeData`, see the comments on the `addBrahmicScheme` and `addRomanScheme` functions.
> +
> +Testing
> +-------
> +
> +Top level `Makefile` Starts a Webserver using `PHP` on Port `9732`.
> +
> +`make` - will start the webserver on port `9732`
> +
> +Open your browser and launch this url `http://127.0.0.1:9732/test/index.html` <http://127.0.0.1:9732/test/index.html> and see that all tests are ok.
>
> I don't see why this is necessary. Can you explain? The localhost/qunit
> approach that exists already seems fine.
> ------------------------------
>
> In README.md
> <https://github.com/sanskrit/sanscript.js/pull/7#discussion_r135688365>:
>
> > @@ -71,3 +71,19 @@ Adding a new scheme is simple:
>      Sanscript.addRomanScheme(schemeName, schemeData);
>
>  For help in creating `schemeData`, see the comments on the `addBrahmicScheme` and `addRomanScheme` functions.
> +
> +Testing
> +-------
> +
> +Top level `Makefile` Starts a Webserver using `PHP` on Port `9732`.
> +
> +`make` - will start the webserver on port `9732`
> +
> +Open your browser and launch this url `http://127.0.0.1:9732/test/index.html` <http://127.0.0.1:9732/test/index.html> and see that all tests are ok.
> +
> +Changelog
>
> No need for changelog, Git tracks that all already.
> ------------------------------
>
> In sanscript/sanscript.js
> <https://github.com/sanskrit/sanscript.js/pull/7#discussion_r135688376>:
>
> > @@ -5,14 +5,29 @@
>   * other Indian languages only incidentally.
>   *
>   * Released under the MIT and GPL Licenses.
> + *
>
> See comment above about changelog.
> ------------------------------
>
> In sanscript/sanscript.js
> <https://github.com/sanskrit/sanscript.js/pull/7#discussion_r135688473>:
>
> >           */
>          bengali: {
>              vowels: 'অ আ ই ঈ উ ঊ ঋ ৠ ঌ ৡ  এ ঐ  ও ঔ'.split(' '),
>              vowel_marks: 'া ি ী ু ূ ৃ ৄ ৢ ৣ  ে ৈ  ো ৌ'.split(' '),
>              other_marks: 'ং ঃ ঁ'.split(' '),
>              virama: ['্'],
> -            consonants: 'ক খ গ ঘ ঙ চ ছ জ ঝ ঞ ট ঠ ড ঢ ণ ত থ দ ধ ন প ফ ব ভ ম য র ল ব শ ষ স হ ळ ক্ষ জ্ঞ'.split(' '),
> -            symbols: '০ ১ ২ ৩ ৪ ৫ ৬ ৭ ৮ ৯ ॐ ঽ । ॥'.split(' '),
> -            other: '    ড ঢ  য '.split(' ')
> +            consonants: 'ক খ গ ঘ ঙ চ ছ জ ঝ ঞ ট ঠ ড ঢ ণ ত থ দ ধ ন প ফ ব ভ ম য় র ল ব শ ষ স হ ল় ক্ষ জ্ঞ'.split(' '),
> +            symbols: '০ ১ ২ ৩ ৪ ৫ ৬ ৭ ৮ ৯ ওঁ ঽ । ॥'.split(' '),
> +            other: '    ড ঢ  য ল'.split(' '),
> +            candra: [''],
> +            accent: ['\u2060', '\u2060', "\u2060", "\u2060"],
> +            combo_accent: ["", "", "", ""]
> +        },
> +		/* Grantha
>
> nit: line up comment with ones below.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sanskrit/sanscript.js/pull/7#pullrequestreview-59125795>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AE2_o-YkLbk4oz_HXt3pRYp4QLpU9vG3ks5sc41VgaJpZM4CvE97>
> .
>

Shreeshrii commented:
If you think it will help, I can close this PR.

You can suggest how to proceed after that.

On 29-Aug-2017 9:42 AM, "ShreeDevi Kumar" <shreeshrii@gmail.com> wrote:

> Arun,
>
> The Readme changes are by Naresh, I had forked his repo for use with
> sanskrit documents.
>
> Not being familiar with git, GitHub I was keeping a changelog within the
> js script for my reference.
>
> I think it would be easiest if you can include the relevant changes in
> your repo and ignore what is unnecessary.
>
> Thanks!
>
>
> On 29-Aug-2017 9:32 AM, "Arun Prasad" <notifications@github.com> wrote:
>
>> *@akprasad* commented on this pull request.
>>
>> Thanks for these changes, and thanks again for your patience.
>> ------------------------------
>>
>> In README.md
>> <https://github.com/sanskrit/sanscript.js/pull/7#discussion_r135688336>:
>>
>> > @@ -71,3 +71,19 @@ Adding a new scheme is simple:
>>      Sanscript.addRomanScheme(schemeName, schemeData);
>>
>>  For help in creating `schemeData`, see the comments on the `addBrahmicScheme` and `addRomanScheme` functions.
>> +
>> +Testing
>> +-------
>> +
>> +Top level `Makefile` Starts a Webserver using `PHP` on Port `9732`.
>> +
>> +`make` - will start the webserver on port `9732`
>> +
>> +Open your browser and launch this url `http://127.0.0.1:9732/test/index.html` <http://127.0.0.1:9732/test/index.html> and see that all tests are ok.
>>
>> I don't see why this is necessary. Can you explain? The localhost/qunit
>> approach that exists already seems fine.
>> ------------------------------
>>
>> In README.md
>> <https://github.com/sanskrit/sanscript.js/pull/7#discussion_r135688365>:
>>
>> > @@ -71,3 +71,19 @@ Adding a new scheme is simple:
>>      Sanscript.addRomanScheme(schemeName, schemeData);
>>
>>  For help in creating `schemeData`, see the comments on the `addBrahmicScheme` and `addRomanScheme` functions.
>> +
>> +Testing
>> +-------
>> +
>> +Top level `Makefile` Starts a Webserver using `PHP` on Port `9732`.
>> +
>> +`make` - will start the webserver on port `9732`
>> +
>> +Open your browser and launch this url `http://127.0.0.1:9732/test/index.html` <http://127.0.0.1:9732/test/index.html> and see that all tests are ok.
>> +
>> +Changelog
>>
>> No need for changelog, Git tracks that all already.
>> ------------------------------
>>
>> In sanscript/sanscript.js
>> <https://github.com/sanskrit/sanscript.js/pull/7#discussion_r135688376>:
>>
>> > @@ -5,14 +5,29 @@
>>   * other Indian languages only incidentally.
>>   *
>>   * Released under the MIT and GPL Licenses.
>> + *
>>
>> See comment above about changelog.
>> ------------------------------
>>
>> In sanscript/sanscript.js
>> <https://github.com/sanskrit/sanscript.js/pull/7#discussion_r135688473>:
>>
>> >           */
>>          bengali: {
>>              vowels: 'অ আ ই ঈ উ ঊ ঋ ৠ ঌ ৡ  এ ঐ  ও ঔ'.split(' '),
>>              vowel_marks: 'া ি ী ু ূ ৃ ৄ ৢ ৣ  ে ৈ  ো ৌ'.split(' '),
>>              other_marks: 'ং ঃ ঁ'.split(' '),
>>              virama: ['্'],
>> -            consonants: 'ক খ গ ঘ ঙ চ ছ জ ঝ ঞ ট ঠ ড ঢ ণ ত থ দ ধ ন প ফ ব ভ ম য র ল ব শ ষ স হ ळ ক্ষ জ্ঞ'.split(' '),
>> -            symbols: '০ ১ ২ ৩ ৪ ৫ ৬ ৭ ৮ ৯ ॐ ঽ । ॥'.split(' '),
>> -            other: '    ড ঢ  য '.split(' ')
>> +            consonants: 'ক খ গ ঘ ঙ চ ছ জ ঝ ঞ ট ঠ ড ঢ ণ ত থ দ ধ ন প ফ ব ভ ম য় র ল ব শ ষ স হ ল় ক্ষ জ্ঞ'.split(' '),
>> +            symbols: '০ ১ ২ ৩ ৪ ৫ ৬ ৭ ৮ ৯ ওঁ ঽ । ॥'.split(' '),
>> +            other: '    ড ঢ  য ল'.split(' '),
>> +            candra: [''],
>> +            accent: ['\u2060', '\u2060', "\u2060", "\u2060"],
>> +            combo_accent: ["", "", "", ""]
>> +        },
>> +		/* Grantha
>>
>> nit: line up comment with ones below.
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/sanskrit/sanscript.js/pull/7#pullrequestreview-59125795>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/AE2_o-YkLbk4oz_HXt3pRYp4QLpU9vG3ks5sc41VgaJpZM4CvE97>
>> .
>>
>

akprasad commented:
Thanks! I'll close this pull request and merge everything in tomorrow.

Shreeshrii commented:
Thanks. Please also make the changes to sanscript.php.

akprasad commented:
Taking a little longer than I expected after merging in another PR. But I'll do this soon (within a week or so).
